cmd/plugin/builder/template/plugintemplates/dotgolangci.txt.tmpl

```diff
@@ -80,7 +85,3 @@ linters:
   # - testpackage
   # - scopelint
   # - wsl
-
-# https://github.com/golangci/golangci/wiki/Configuration
-service:
-  golangci-lint-version: 1.38.0 # use the fixed version to not introduce new linters unexpectedly
```

Comment:
golangci-lint recommends using the same version as the actual golangci-lint installed, so removed this config, which seemed unnecessary.